PROVISIONING-METAL.md

```diff
@@ -265,6 +265,35 @@ Older networking configuration versions (such as `1` or `2`) are supported in ne
 | Version 2                     | [v1.10.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.10.0) |
 | Version 3                     | [v1.12.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.12.0) |
 
+#### Validate network configuration
+
+There is a `make` target `netdog-parse` to validate that network configuration files parse correctly.
```

Comment:
no there isn't

Reply:
Whoops, I updated the command, not the docs.